packages/jaeger-ui/src/model/link-patterns.tsx

```diff
@@ -110,6 +110,22 @@ export function getParameterInAncestor(name: string, span: Span) {
     }
     currentSpan = getParent(currentSpan);
   }
+
+  return undefined;
+}
+
+export function getParameterInTrace(name: string, trace: Trace | undefined) {
```

Comment:
looks like an overlap with `computeTraceLink`, can we find a way to do this only once?

Reply:
Addressed the feedback, let me know if it looks OK.